src/chart/components/chart/candle.functions.ts

```diff
@@ -51,3 +56,5 @@ export const deleteCandlesIndex = (candles: Array<Candle>) => {
 		candle.idx = undefined;
 	});
 };
+
+export const isCandle = (value: unknown): value is Candle => value !== undefined;
```

Comment:
the function does not correspond to its intended purpose, rewrite it pls

Reply:
I can pass any value in this function, except undefined, and if will return true, If you want to create such function, you should check full type guard for every property. But If you want to filter array from empty values, you don't need this function